Makefile

```diff
@@ -96,9 +96,11 @@ load-fixtures:
 	$(MAKE) load-synthetic-suppression-data
 	$(MAKE) load-fixtures-ssas
 	
-	# Add ALR data for ACO under test. Must have attribution already set.
-	$(eval ACO_CMS_ID = A9994)
-	docker-compose run api sh -c 'bcda generate-synthetic-alr-data --cms-id=$(ACO_CMS_ID) --alr-template-file ./alr/gen/testdata/PY21ALRTemplatePrelimProspTable1.csv'
+	# Add ALR data for ACOs under test. Must have attribution already set.
+	$(eval ACO_CMS_IDS := A9994 A9996 E9996 V996) 
```

Comment:
So sorry about doing a 180 on this.

Since we may make additional changes to the synthetic data, could we just limit this to one ACO. Maybe A9994?